packages/builder/src/stores/builder/queries.ts

```diff
@@ -26,6 +26,7 @@ interface BuilderQueryStore {
 
 interface DerivedQueryStore extends BuilderQueryStore {
   selected?: Query
+  listByType?: Query[]
```

Comment:
Is this used? I cannot see it being used anywhere

Reply:
Nope, nice catch! 👍